src/nc_util.c

```diff
@@ -75,6 +78,24 @@ nc_set_reuseaddr(int sd)
     return setsockopt(sd, SOL_SOCKET, SO_REUSEADDR, &reuse, len);
 }
 
+int
+nc_set_reuseport(int sd)
+{
+    int reuse;
+    socklen_t len;
+
+    reuse = 1;
+    len = sizeof(reuse);
```

Comment:
I think this should be replaced with `#ifdef SO_REUSEPORT`, which would cover all of the OS cases

Reply:
Thanks, fixed, it was quite an old commit :)